src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/advice/UserProfileControllerAdvice.java

```diff
@@ -40,6 +40,9 @@
     @Value("${resendInterval}")
     private String resendInterval;
 
+    @Value("${logging-component-name}")
+    protected static String loggingComponentName;
```

Comment:
make this variable to private everywhere if not used by child class

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/395/commits/776a4b17788086cb843cb412cd6519e17bc09983